molecule/ksql-scale-up/converge.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Verify - control_center
+  hosts: control_center
+  gather_facts: false
+  tasks:
+    - name: Check line ksql cluster2
+      import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/confluent-control-center/control-center-production.properties
+        property: confluent.controlcenter.ksql.ks2.url
+        expected_value: http://ksql2:8088
```

Comment:
why do we have ksql1 and ksql2 groups ?

Reply:
Since this is a KSQL Scale up scenario, we are testing with 2 different clusters of KSQL. ks1 and ks2.
We are adding one node ksql3 to one group and ksql4 to another group.
Idea here is to ensure the capability that while scaling, it's not messing up with the KSQL (Logical cluster groups) groups, and properly getting added to the desired group.